django_react_proj/users-fe/src/components/UserList/UserList.js

```diff
@@ -19,6 +22,27 @@ const Actions = ({ user, resetState }) => {
     });
     resetState();
   };
+  const moveToPreviousDay = async () => {
+    const result = isSameDay(new Date(), new Date(user.startTime));
+    if (result === false) {
```

Comment:
`!result`

Reply:
Value of result must be false to move task to previous day. If result is true (that means planned day of task is today) you can't move it to previous day.